### PR TITLE
Some more front matter maintenance

### PR DESCRIPTION
**Pull request by marians**
This pR attempts to further improve our front matter.

As the user_questions length limit of 80 characters turns out to be very difficult to reach, I am increasing it to 100.

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/search/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Search the documentation
-description: Full text search for the Giant Swarm documentation and changelogs.
+description: Full text search for the Giant Swarm documentation, also coverign the REST API documentation and release notes.
```
Comment (Oshratn) [inline src/content/search/_index.md:3]:
```suggestion
description: Full text search for the Giant Swarm documentation, also covering the REST API documentation and release notes.
```

**Review by Oshratn — APPROVED** [review]
Apart from the typo LGTM

File: src/content/ui-api/gsctl/cluster-definition.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ menu:
 aliases:
   - /reference/cluster-definition/
 user_questions:
-  - What are additional fields I need to fill out for node pool clusters
+  - What are additional fields I need to fill out for node pool clusters?
```
Comment (stone-z) [inline src/content/ui-api/gsctl/cluster-definition.md:12]: (could also say "required")
```suggestion
  - What additional configuration options are available for node pool clusters?
```

**Review by stone-z — APPROVED** [review]
LGTM - one suggestion

**Pull request merged**